NewForce-Cohort-9/tabloid-full-stack-api-all-stars
# Ticket #13: View All Tags

## Description
When an admin is on the Tag Management page they should see a list of all tags listed alphabetically. [Trello Ticket #13](https://trello.com/c/rgj2sXTF/13-13-view-all-tags)

## Type of change

## Testing Instructions

1. `git add` and `git commit` whatever you're currently working on 
2. `git checkout main`
3. `git fetch --all`
4. git checkout `bp-beginning`
5. If you don't see the most recent version of my files, `git pull origin bp-beginning`
6. If you haven't created the Tabloid database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-api-all-stars/blob/main/SQL/01_Tabloid_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-api-all-stars/blob/main/SQL/01_Tabloid_Create_DB.sql)
9. Click the execute with debugger solid green button in purple Visual Studio to launch the api
10. In your terminal, launch the program within the root folder with the command `npm start`


### Then:
* In the browser, please go to Login on the nav bar and log in with this email: ```foo@bar.com```
* This should redirect you back to the home page.
* Confirm that Tag Management shows the nav bar.
* Select Tag Management to be redirected to the Tag Management page to see a list of all tags ordered alphabetically.


## Checklist:
